autode/wrappers/XTB.py

```diff
@@ -50,29 +50,31 @@ def print_cartesian_constraints(self, inp_file, molecule):
         if molecule.constraints.cartesian is None:
             return None
 
-        constrained_atom_idxs = [i + 1 for i in molecule.constraints.cartesian]
-        list_of_ranges, used_atoms = [], []
-
-        for i in constrained_atom_idxs:
-            atom_range = []
-            if i not in used_atoms:
-                while i in constrained_atom_idxs:
-                    used_atoms.append(i)
-                    atom_range.append(i)
-                    i += 1
-                if len(atom_range) in (1, 2):
-                    list_of_ranges += str(atom_range)
-                else:
-                    list_of_ranges.append(f"{atom_range[0]}-{atom_range[-1]}")
+        atom_idxs = list(
+            sorted(int(i) + 1 for i in molecule.constraints.cartesian)
```

Comment:
I am wondering if it would be useful to have autode log a warning if there is only one constrained atom in cartesian. (which does not really make sense). xTB seems to accept if one atom is mentioned in `$constrain`.

Reply:
I don't see anything bad happening if only one atom is constrained so not sure it should be a warning. Going to leave as is
(thanks for the review 👍🏼)